ethexe/db/src/database.rs

```diff
@@ -139,6 +139,14 @@ impl Database {
         }
     }
 
+    pub fn new_in_memory(router_address: [u8; 20]) -> Self {
```

Comment:
also use `from_one` to keep it the same object?
```suggestion
    pub fn memory(router_address: [u8; 20]) -> Self {
```